SadConsole.Host.SFML/GameTexture.cs

```diff
@@ -95,10 +104,27 @@ public Color[] GetPixels()
             return colors;
         }
 
+        /// <inheritdoc />
+        public void SetPixels(Color[] colors)
+        {
+            if (colors.Length != Size) throw new ArgumentOutOfRangeException("Pixels array length must match the texture size.");
+
+            byte[] pixels = new byte[Size * 4];
```

Comment:
Can commit this update? I would like a note here. I think you could convert this to a byte array and avoid copying splatting the color data out into the byte array.
```suggestion
            // TODO: Check if we can cast via MemoryMarshal.Cast as long as Color is packed appropriately
            byte[] pixels = new byte[Size * 4];
```